src/main/kotlin/step3_4/view/InputView.kt

```diff
@@ -0,0 +1,27 @@
+package step3_4.view
+
+import step3_4.validator.CarNameValidator
+import step3_4.validator.RoundNumberValidator
+
+object InputView {
+
+    fun processEnteredCarInformation(): List<String> {
+        val input = readlnOrNull().orEmpty()
```

Comment:
```suggestion
        val input = readln()
```
으로 `String?`을 orEmpty로 재처리하는 것이 아닌 바로 `String`로 반환해도 되지 않을까요?

Reply:
ed15de3e5857be193a8ccebc386aa6e1db45a517 반영하였습니다